gulpfile.js

```diff
@@ -191,7 +191,11 @@ gulp.task('extension:copy', () =>
   es
     .merge(
       gulp.src(
-        path.join(paths.extensions.root, '**', '@(vss-extension.json|extension-icon.png|*.md)')
+        path.join(
+          paths.extensions.root,
+          '**',
+          '@(vss-extension.json|extension-icon.png|*.md|*.html)'
```

Comment:
I don't see any html file added to the extension, so why do you need to add that?

Reply:
Ah thanks it's a leftover from one of my early testing!